_includes/configuration/toolbar-mode.md

```diff
@@ -0,0 +1,98 @@
+## toolbar_mode
+{% if site.productminorversion < 6.1 %}
+<a class="anchor" id="toolbar_drawer"></a>
+> **Note**: The `toolbar_drawer` option was deprecated with the release of {{site.productname}} 5.2. The `toolbar_drawer` option was renamed `toolbar_mode`. This change was made to reflect the range of settings available for this option.
```

Comment:
`renamed to`?

Reply:
either should be fine.

Reply:
I prefer "renamed to" as it's more correct to me...

Reply:
updated